CONTRIBUTING.md

```diff
@@ -74,6 +77,22 @@ Cluster API uses [GitHub milestones](https://github.com/kubernetes-sigs/cluster-
 - Dates in a release are approximations and always subject to change.
 - `Next` milestone is for work that has been triaged, but not prioritized/accepted for any release.
 
+## Proposal process (CAEP)
+
+The Cluster API Enhacement Proposal is the process this project uses to adopt new features, or changes to the APIs.
+
+The [template](https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/proposals/YYYYMMDD-template.md), and accepted proposals live under [docs/proposals](https://github.com/kubernetes-sigs/cluster-api/tree/master/docs/proposals).
+
+- Proposals or requests for enhacements (RFEs) MUST be discussed in an issue first.
+  - Issues can be placed on the roadmap during planning if there is one or more folks
+    that can dedicate time to writing a CAEP and/or implementating it after approval.
+- A proposal SHOULD be introduced and discussed during the weekly community meetings,
+  [Kubernetes SIG Cluster Lifecycle mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle),
+  or [discuss forum](https://discuss.kubernetes.io/c/contributors/cluster-api/).
```

Comment:
I'd like to see us do a better job of broadcasting proposals than we do currently. If someone isn't watching the repository and attending all the meetings it is difficult to stay on top of proposed changes. I'm not necessarily saying it should be the responsibility of the person creating the proposal, but as a project we should do a better job of updating the mailing list and discuss forum know about proposals with a decent lead time for folks to review and provide feedback.

Reply:
How would you like for this to change? We can make it maintainer's responsibilities?

Reply:
I think making it the maintainer's responsibility makes sense to me, the main goal to allow folks to more easily participate and keep track of proposed changes without having to fully drink from the firehose.

Reply:
today for the kubernetes community, the goto repo for proposals is `k/enhancements` at some point we might want have `CAEPs` live next to `KEPs` under the sig cluster-lifecycle folder. I think this would also broaden the audience of the project. thoughts ?

Reply:
there is a folder for CAPI KEPs here:
https://github.com/kubernetes/enhancements/tree/master/keps/sig-cluster-lifecycle/clusterapi

we should PR a README.md to that folder to point to the k-sigs/cluster-api folder for CAEPs.

if CAEPs start living in k/e, at minimum we would have to auto-apply the `tracked/out-of-tree` label to the CAPI folder to try to reduce the noise for the release team, who normally track k8s cadence proposals.

Reply:
k/enhancements is it's own firehose, not saying we couldn't standardize on using it, I'm just not sure if it would achieve better community visibility than we have today.

Reply:
`k/e` Cluster API proposal seems very out of date 😬, we probably need to add a minimum of text that says we've our own process. I'm in favor of keeping the process in house though, rather than using the broader process

Reply:
I don't see a benefit of using k/e, as maintainers of the k/e process aren't (understandably) going to be invested in process improvements for us.

Reply:
> Cluster API proposal seems very out of date

`+`

> there is a folder for CAPI KEPs here...(snip)...we should PR a README.md to that folder (in k/e)

i will log a ticket for these two.

Reply:
https://github.com/kubernetes-sigs/cluster-api/issues/3791

Reply:
sounds good to me, I was mainly asking for discoverability